pygotham/admin/utils.py

```diff
@@ -51,7 +48,7 @@ def menu_link(name, endpoint, authenticated):
 
 
 def model_view(model, name, category=None, **kwargs):
-    """Return a subclass of :class:`~flask.ext.admin.contrib.sql.ModelView`.
+    r"""Return a subclass of :class:`~flask.ext.admin.contrib.sql.ModelView`.
```

Comment:
Because of the asterisks.

Long-term I'd like to switch to Google-style docstrings. But I figure that's a big enough change to warrant its own pull request.